DokuwikiXMLExport.php

```diff
@@ -37,6 +38,18 @@ class DokuwikiXMLExport
     const CATEGORY_KEY = 'cat';
 
     /**
+     * This value is the key for a dummy property.
+     * Hotfix workaround for a bug @ FINDOLOGIC
+     */
+    const PROPERTY_DUMMY_KEY = 'dummy';
+
+    /**
+     * This value is the value for a dummy property.
+     * Hotfix workaround for a bug @ FINDOLOGIC
+     */
+    const PROPERTY_DUMMY_VALUE = array('dummy');
```

Comment:
Properties do not support multiple values so you can just use a single value here.

Reply:
@georgms 

`libflexport` only accepts an array here.
The reason for this is to support multible usergroups.
If I would set multible values e.g. `const PROPERTY_DUMMY_VALUE = array('dummy', 'dummy2');`
The result would look like this:
![selection_097](https://user-images.githubusercontent.com/20555112/30549643-969fc412-9c95-11e7-8c75-2ded3ffd3e2c.png)
If I wouldn't use an array here there wouldn't be any properties at all.

Please re-review this, thanks!

Reply:
Got it, LGTM!